About.html

```diff
@@ -35,7 +35,7 @@ <h2>About Adegboyega</h2>
                 <a href="#"><img src=".\img\GitHub-Mark-32px.png" alt=""></a>
                 <a href=""><img src=".\img\iconfinder_06-facebook_104498.png" alt=""></a>
                 <a href=""> <img src=".\img\iconfinder_Instagram_1298747.png" alt=""></a>
-                <a href=""><img src=".\img\iconfinder_Linkedin_Color_1435161.png" alt=""></a>
+                <a href=""><img src="  .\img/iconfinder_square-linkedin_317725.png" alt=""></a>
```

Comment:
You are missing an `alt` attribute for the image element.

Reply:
Seen. will add alt values and push today.

Reply:
I've fixed all alt values now